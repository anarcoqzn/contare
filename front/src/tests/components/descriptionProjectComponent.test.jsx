import DescriptionProject from '../../components/descriptionProjectComponent/descriptionProjectComponent';

it('should render correctly', () => {
    const wrapper = shallow(<DescriptionProject />);
    expect(wrapper).toMatchSnapshot();
});