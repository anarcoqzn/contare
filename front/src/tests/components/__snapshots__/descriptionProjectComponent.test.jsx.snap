// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<[object Object]>
  Lorem ipsum ultrices velit faucibus donec adipiscing nulla arcu consectetur velit etiam donec, aliquet sapien purus quisque tempor auctor habitant imperdiet lacus at mollis, nec sit venenatis hendrerit primis taciti facilisis aenean mattis lorem vitae. cras integer morbi id nunc per etiam class per, commodo turpis morbi iaculis at lectus sociosqu, fringilla magna sociosqu dictumst non massa taciti. cursus potenti nibh in dictum torquent cras sagittis molestie, scelerisque curae elementum at diam egestas placerat curabitur, posuere iaculis ultrices congue dolor integer inceptos. interdum maecenas sit neque bibendum nec quisque massa scelerisque, dui nostra in blandit conubia accumsan suspendisse, eros aliquam massa justo sagittis ullamcorper feugiat.
</[object Object]>
`;
